DeutschJozsaAlgorithm/ReferenceImplementation.qs

```diff
@@ -329,8 +323,8 @@ namespace Quantum.Kata.DeutschJozsaAlgorithm {
             }
             
             // before releasing the qubits make sure they are all in |0⟩ state
-            ResetAll(x);
-            Reset(y);
+            // ResetAll(x);
```

Comment:
One more comment: you can remove these Resets altogether (and the comment before them as well), since the don't make for helpful comments.